### ISSUE TITLE
[calendar] new feature: support username like ldap addressbook

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#298]
```
Hi,

@ my company a person my have more than one e-mail address and he/she can login 
to roundcube with each of them.
So I wanted to get the same calendar independent of the loginname he is using.

Because the addresses are always like user.name@domain.tld I've implemented a 
functionality like the ldap addressbook of roundcube is providing (%u  - The 
username prior to the '@').

Attached are the patches for calendar.php (username_su-calendar.patch) and 
program/backend/caldav.php (username_su-backend_caldav.patch) needed to 
implement this feature and created against revision 70.
The replacement string is %su. It will if available always be replaced with the 
username prior to the '@'.

I hope you'll add this feature.


Kind regards,
Andrwe
```

Original issue reported on code.google.com by `lord.web...@googlemail.com` on 21 Feb 2012 at 5:12

Attachments:
- [username_su-backend_caldav.patch](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-298/comment-0/username_su-backend_caldav.patch)
- [username_su-calendar.patch](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-298/comment-0/username_su-calendar.patch)

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 27 Feb 2012 at 8:13
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in bundle v1.7
```

Original comment by `rol...@liebl.ath.cx` on 1 Mar 2012 at 4:58
- Changed state: **Fixed**
